app/components/agenda/agendaitem/agendaitem-postponed.js

```diff
@@ -0,0 +1,97 @@
+import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
+import { inject as service } from '@ember/service';
+import { task } from 'ember-concurrency';
+import addWeeks from 'date-fns/addWeeks';
+import { PAGE_SIZE } from 'frontend-kaleidos/config/config';
+
+/**
+ * @argument subcase
+ * @argument meeting
+ * @argument agendaActivity
+ */
+export default class AgendaitemPostponed extends Component {
+  @service store;
+  @service agendaService;
+
+  @tracked canPropose = false;
+  @tracked modelsForProposedAgenda;
+  @tracked newMeeting;
+
+  constructor() {
+    super(...arguments);
+    this.loadProposedStatus.perform();
+    this.loadProposableMeetings.perform();
+  }
+
+  @task
+  *loadProposedStatus() {
+    // We have to check if the proposed agendaitem is already proposed for a new meeting before showing the button
+    // we have to generate a link to the latest meeting (not the next if multiple).
```

Comment:
I don't understand this comment since nothing is done with `meeting` here

Reply:
Note sure what part of the comments you mean.
_We have to check if the postponed agendaitem is already proposed for a future meeting_
Could rephrase this "We have to check if there is a more recent agenda-activity. If there is none, we can repropose to a new meeting"

_we do use the meeting further down to generate a link._
we do that further down

_filtering on meeting.plannedStart will not find any if reProposing happened before the meeting "started" at 10 am_
Added this comment because I tried filtering on `meeting.plannedStart` in the query with `activity.startDate`, but that meant you could no longer repropose an agendaitem that was added to a friday meeting past 10am so dropped that idea.

Reply:
This is the part I was missing :+1: 

> _ we have to generate a link to the latest meeting (not the next if multiple)._
we do that further down.
We could technically show the link to the next agenda but in the spec was said we go the last one instead. When 1 subcase has been reproposed multiple times.